incubator/hnc/cmd/manager/main.go

```diff
@@ -24,6 +24,7 @@ import (
 	prom "github.com/prometheus/client_golang/prometheus"
 	"go.opencensus.io/stats/view"
 	corev1 "k8s.io/api/core/v1"
+	"k8s.io/apiextensions-apiserver/pkg/apis/apiextensions/v1beta1"
```

Comment:
Maybe make a note that this should be changed to v1 when we only need to support 1.17 and higher?

Reply:
Done.

Reply:
Do we have a stated goal of our supported k8s version? Are we by default following k8s (3 versions back, I know there is work being done for LTS as well) or something different? May be good to have an official statement in docs somewhere.

Reply:
Just chatted with Adrian about supported k8s version. 

From Adrian, we are not completely sure what k8s versions we should support at this stage. 

One idea is that we should at least support whatever k8s versions that are supported by GKE (e.g., my GKE cluster is using 1.13). For the other k8s versions, we may not test if it is compatibility with HNC. 

Another idea is that we can support k8s versions that are released in the past 18 months (e.g., from 1.11 to 1.17). 

Feel free to let us know if you have suggestions!